yt/visualization/tests/test_color_maps.py

```diff
@@ -0,0 +1,51 @@
+import os
+import shutil
+import tempfile
+import unittest
+
+import matplotlib
+import numpy as np
+from nose.tools import assert_raises
+
+from yt import show_colormaps, make_colormap
+from yt.testing import assert_equal, assert_almost_equal
+
+matplotlib.use('Agg')
```

Comment:
I'm not sure this will work as you're expecting. You can only change the matplotlib backend once, so if another test does this then this line won't work. I think it would also set the backend for all future tests that use matplotlib, which isn't what we want either.

It might be better to create a matplotlibrc file on travis and appveyor that sets `'Agg'` as the default backend, and add a dectorator to `yt.testing` that checks to see what the backend is and only lets a test run if the backend is agg, something like `@requires_backend('Agg')`. That way if people run the tests on their laptops and Agg isn't the default backend, this test will still pass and `show_colormaps()` won't pop up an interactive matplotlib plot and block the test runner.

If what I just said isn't clear let me know and I can try to explain in a little more detail.

Reply:
done